product_docs/docs/eprs/7/10_appendix/03_miscellaneous_xdb_processing_topics/01_publications_and_subscriptions_server_conf_options/snapshot_connection_retry.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: "Snapshot replication connection retry"
+---
+
+When a connection failure with the subscription database occurs during a snapshot replication, 
+the Replication Server automatically attempts to reconnect to it. 
+The goal is that the snapshot replication completes without skipping any tables. When the connection is reestablished, 
+Replication Server resumes the snapshot replication procedure and performs the data copy for the remaining tables.
```

Comment:
I don't think the last sentence is quite right. The goal of the new MTK connection retry capability is ensure that the table that was being migrated when the connection to the target database is lost is fully migrated if the connection can be reestablished prior to performing the data copy on the remaining tables.  I recommend the following update to the last sentence:

> When the connection is reestablished, Replication Server restarts or resumes the snapshot replication procedure for the table that was being migrated when the connection was lost and then performs the data copy for the remaining tables.

@anwar-sahib @zahid75 Does this recommended update capture the behavior more accurately?

Reply:
@mlewandowski-edb  In agreement with your modified contents, the original contents are a bit confusing as it sounds like EPRS will resume Snapshot for the subsequent tables.